rclpy/rclpy/logging_service.py

```diff
@@ -12,17 +12,25 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Union
+
 from rcl_interfaces.msg import LoggerLevel, SetLoggerLevelsResult
 from rcl_interfaces.srv import GetLoggerLevels
 from rcl_interfaces.srv import SetLoggerLevels
 import rclpy
+from rclpy.impl.logging_severity import LoggingSeverity
```

Comment:
I think this should be in the `TYPE_CHECKING` block, as we don't need it otherwise.